config/v1/types_feature.go

```diff
@@ -81,6 +83,8 @@ var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
 			"ExperimentalCriticalPodAnnotation", // sig-pod, sjenning
 			"RotateKubeletServerCertificate",    // sig-pod, sjenning
 			"SupportPodPidsLimit",               // sig-pod, sjenning
+			"CSIBlockVolume",                    // sig-storage, j-griffith
+			"BlockVolume",                       // sig-storage, j-griffith
```

Comment:
@sjenning @childsb I'm not as opposed to doing this here, in 4.1. Your thoughts?

Reply:
Actually, is `BlockVolume` needed at all? Isn't this already on in 4.1?  (So at best this is a 1-line patch)

Reply:
It's still listed as Alpha in the upstream docs, I'll get latest OCP deployed and confirm but IIRC it's disabled (I very well could be mistaken though)

https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Reply:
There are 2 lines for `BlockVolume` in that doc. If you see it says - `1.13 - Beta` and since 4.1 includes 1.13, Blockvolume should be enabled.